DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -19,8 +19,10 @@ public RegistrationDataService(IDbConnection connection)
             this.connection = connection;
         }
 
-        public string RegisterDelegate(DelegateRegistrationModel delegateRegistrationModel)
+        public (string candidateNumber, bool approved) RegisterDelegate(DelegateRegistrationModel delegateRegistrationModel)
         {
+            var approved = false;

```

Comment:
Could the approved flag be taken from the delegateRegistrationModel - I think Dan has added it in his MR, but yours will be merged first? (you could just set it to false in the constructor)

https://github.com/TechnologyEnhancedLearning/DLSV2/pull/339/files

+ then we don't need to return approved from the dataService (as that flag is specified in the service layer and is not dependent on creating a delegate entry in the db)